python3-ec2imgutils.spec

```diff
@@ -33,6 +33,7 @@ Requires:       python3-paramiko
 BuildRequires:  python3-boto3 >= 1.18.7
 BuildRequires:  python3-dateutil
 BuildRequires:  python3-setuptools
+BuildRequires:  python3-rpm-macros
```

Comment:
Please use `python-rpm-macros`. `python3-rpm-macros` is just a `Provides` and as such if someone searches for `python3-rpm-macros` in the Build Service nothing will show up which will lead to an unnecessary goose chase.

Reply:
Rebased.